#!/usr/bin/env python

PACKAGE="pcl_assembler"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#old cfg file used for DoN based clustering

gen.add("debug", bool_t,   0, "debug: ",   False)
gen.add("scale_1",         double_t, 0, "scale1: ", 0.02,     0.0, 10.0)
gen.add("scale_2",        double_t, 0, "scale2: ",     0.9,     0.0, 10.0)
gen.add("seg_radius",        double_t, 0, "segradius: ",          0.1,     0.0, 10.0)
gen.add("threshold_",        double_t, 0, "threshold: ",          0.2,     0.0, 10.0)


exit(gen.generate(PACKAGE, "pcl_normal_filter", "ClusteringParam"))
